lib/matplotlib/tests/test_table.py

```diff
@@ -194,3 +195,34 @@ def test_table_cells():
     # properties and setp
     table.properties()
     plt.setp(table)
+
+
+@check_figures_equal(extensions=["png"])
+def test_table_bbox(fig_test, fig_ref):
+    data = [[66386, 174296],
+            [58230, 381139]]
+
+    colLabels = ('Freeze', 'Wind')
+    rowLabels = ['%d year' % x for x in (100, 50)]
+
+    cellText = []
+    yoff = np.zeros(len(colLabels))
+    for row in reversed(data):
+        yoff += row
+        cellText.append(['%1.1f' % (x/1000.0) for x in yoff])
```

Comment:
I got a little lost in this.  Could `cellText` be defined directly?

Reply:
I modified this to be a bit more straightforward (just copied the code from the bug report, but naturally it has nothing to do with that).